### ISSUE TITLE
Most Recent TV Episodes

### ISSUE THREAD

**Issue by ryanmccombe** [bbc/simorgh#8304]
https://jira.dev.bbc.co.uk/browse/WSMEDIA-226

**User Story:**
- As a WS visitor, Bard
- I want to be able to access onDemand TV programmes for a brand
- so that I can watch the latest news programmes online to programmes after they have been broadcast

**Description:** 
Add the "Recent Episosdes" component to the ondemand TV episode and brand pages. 
The recent episodes are in the AReS endpoint under relatedContent.group.promos
https://ares-api.api.bbci.co.uk/api/ws-media/tv/afrique/bbc_afrique_tv/programmes/w13xttmz
AReS returns up to 8 episodes if they are available

**Acceptance Criteria:**
Ability to view and select recently available ondemand TV programmes for a TV brand

The promo displays a h2 that Recent "Recent Episodes" in the language of the service (we will use the equivalent PAL translation)  

The promo displays at least 1 episode and max 4 episodes available to be played within a brand 

User can click the episode promo to visit the episode page

The promo does not display the episode that is currently available to play on the page

The promo displays:
- The episode image
  - The episode image must have alt-text (this is not in the data so must be hard coded to eg "BBC News Afrique")
- Brand Title
- Release Date in the service timestamp
- Duration of the programme in the service numerals
- Link to the episode page

**Designs:**
https://zpl.io/V109zjQ

**Screen Reader UX:**
https://zpl.io/bAMWZ8x

**a11y A/C:**
**Recent Episodes - Landmark**
- Given I use a screen reader
- When I navigate to ‘Recent episodes’ 
- Then a complementary landmark must be announced
- And this must not be nested within another landmark e.g. the main landmark
- And this must be announced with a label e.g. Recent episodes
- And the label must be unique to the page

**Recent Episodes - h2**
- Given I use a screen reader
- When I navigate to ‘Recent episodes’ 
- Then a heading level 2 must be announced as the first piece of content
- And the h2’s announced text should be the same as the complementary landmark’s label for this section
- And the h2 must be unique to the page
- And the section must not include any other h2’s

**List of promos within section**
- Given I use a screen reader
- When I view a section with more than one promo 
- And I navigate to the first promo
- Then it should be announced as a list item within a list

**Promo image**
- Given I use a screen reader
- When I view a promo with an image 
- And I navigate to the promo image
- Then the image alt text must be announced
- and the alt text will be the default text for that service e.g. "BBC News Afrique" 

**Recent Episodes - list item link without Episode Title**
- Given there is no episode title in the recent episode promo data 
- and I am using a screenreader
- When I navigate to the link in a ‘Recent episodes’ promo
- Then the announced link text is “Video, [Brand Title], [Release Date], Duration [duration]” in the language of the service
- And the media indicator is not announced on it’s own e.g. "video"
- And there are no heading semantics announced 

**Recent Episodes - list item link with Episode Title**
- Given there is episode title in the recent episode promo for an episode
- and I am using a screenreader
- When I navigate to the link for that episode in the ‘Recent episodes’ promo 
- Then the announced link text is “Video, [Brand Title], [Episode title], Duration [duration]”  in the language of the service
- and when the user navigates to the timestamp
- then the timestamp data is announced 
- And the media indicator is not announced on it’s own e.g. "video" 
- And there are no heading semantics announced 

Full Promo a/c is here for further reference: https://docs.google.com/document/d/1AWDb6QT75uNm9vssoWQvcajv5I8R_XK6oJd631gmyOM/edit?usp=sharing

**Comment by LilyL0u** [comment]
Manual testing complete. Will move this issue to done when the e2e tests have run and passed, and when live checks have been done.

**Comment by RichardPK** [comment]
It's alive! Closing because we've released
